Dockerfile

```diff
@@ -7,9 +7,11 @@ ENV TARGET_APP $TARGET_APP
 # Build target dependencies #
 #############################
 FROM base AS dependencies
-COPY package.json yarn.lock .yarnrc ./
+# Yarn
+RUN yarn set version berry
+COPY package.json yarn.lock .yarnrc.yml ./
 COPY schema.prisma .
-RUN yarn --production
+RUN yarn workspaces focus --all --production
```

Comment:
We can copy `.yarn` in order to send the initial cache to the docker image

Not super important but prices becomes faster.

Reply:
Yep, added now!